Mathlib/NumberTheory/NumberField/Basic.lean

```diff
@@ -420,6 +420,10 @@ theorem coe_ringOfIntegersEquiv (z : 𝓞 ℚ) :
   obtain ⟨z, rfl⟩ := Rat.ringOfIntegersEquiv.symm.surjective z
   simp
 
+theorem ringOfIntegersEquiv_symm_coe (x : ℤ) :
```

Comment:
This isn't really consistent with the name of the previous theorem. What about `ringOfIntegersEquiv.apply_coe` and `ringOfIntegersEquiv.symm_apply_coe`?